comp/core/status/render_helpers.go

```diff
@@ -291,7 +289,7 @@ func formatTitle(title string) string {
 	title = strings.Join(words, " ")
 
 	// Capitalize the first letter
-	return cases.Title(language.English, cases.NoLower).String(title)
+	return strings.ToUpper(string(title[0])) + title[1:]
```

Comment:
This panics if you call with an empty string.

Reply:
I think an empty string would be an error, BUT I think we might as well avoid panics, fixed !